test_utils/__init__.py

```diff
@@ -116,4 +118,35 @@ def set_jwt_cookie(self, roles_and_contexts=[]):
         })
         jwt_token = generate_jwt_token(payload)
 
-        self.client.cookies[jwt_cookie_name()] = jwt_token
\ No newline at end of file
+        self.client.cookies[jwt_cookie_name()] = jwt_token
+
+
+class APITestWithMockedDiscoveryApiClient(APITest):
```

Comment:
nit: should we make this name more generic? i.e. APITestWithMocks in case we ever add more mocks to it?.

Reply:
i figured i'd be explicit at first cause it was just one thing. something something zen of python 🤷 -- i can change it though if you'd like

Reply:
gonna hold off on this for now until we update the mock